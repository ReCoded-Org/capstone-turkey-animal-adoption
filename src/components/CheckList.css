.checkListLogo {
    background-color:#FF7777;
    width: 100%;
    max-width: 65px;
    height: 65px;
    border-radius: 65px;
    display: flex;
    align-items: center;
    justify-content: center;
    position: absolute;
    top: -30px;
}

.catLogo {
    width: 100%;
    max-width: 50px;
    
    
}

.checkListText {
    color:#7D7D7D;
}

.checkListButton {
    border-radius: 25px;
    background-color:#FF7777;
    color:#FFFFFF;
    border:none;
    outline: none;
    padding: 0.5rem 1rem;
}

.checkList {
    background-color: #F5F5F5;
    border-radius: 10px;
    display: flex;
    flex-direction: column;
    align-items: center;
    margin-bottom: 5rem;
    padding: 2rem 0;
    box-shadow: 0 4px 4px -2px #7D7D7D;
}
