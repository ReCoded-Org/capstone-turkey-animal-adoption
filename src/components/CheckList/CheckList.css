.checkListLogo {
  background-color: #ff7777;
  width: 100%;
  max-width: 65px;
  height: 65px;
  border-radius: 65px;
  display: flex;
  align-items: center;
  justify-content: center;
  position: absolute;
  top: -30px;
}

.catLogo {
  width: 100%;
  max-width: 50px;
}

.checkListText {
  color: #7d7d7d;
}

.checkListButton {
  border-radius: 25px;
  background-color: #ff7777;
  color: #ffffff;
  border: none;
  outline: none;
  padding: 0.5rem 1rem;
}

.checkList {
  background-color: #f5f5f5;
  border-radius: 10px;
  display: flex;
  flex-direction: column;
  align-items: center;
  margin-bottom: 5rem;
  padding: 2rem 0;
  box-shadow: 0 4px 4px -2px #7d7d7d;
}
