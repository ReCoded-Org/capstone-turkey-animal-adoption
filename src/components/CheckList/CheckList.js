import React from "react";
import catLogo from "../../checklistLogo/catLogo.png";
import "./CheckList.css";
import { Col } from "react-bootstrap";
import { useTranslation } from "react-i18next";

const Checklist = ({ text, title, link }) => {
  const { t } = useTranslation();

  return (
    <Col lg={4}>
      <div className="checkList">
        <div className="checkListLogo">
          <img className="catLogo" src={catLogo} />
        </div>
        <h2>{title}</h2>
        <p className="checkListText">{text}</p>
        <a href={link} target="_blank">
          <button className="checkListButton">{t("checklistButton")}</button>
        </a>
      </div>
    </Col>
  );
};

export default Checklist;
