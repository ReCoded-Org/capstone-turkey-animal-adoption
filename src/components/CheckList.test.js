import CheckList from "./CheckList";
import React from "react";
import renderer from "react-test-renderer";

const data = {
  text: "Go to Checklist",
  title: "Cat Adoption Checklist",
  link: "denemeLink",
};

it("renders correctly", () => {
  const tree = renderer
    .create(<CheckList text={data.text} title={data.title} link={data.link} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
