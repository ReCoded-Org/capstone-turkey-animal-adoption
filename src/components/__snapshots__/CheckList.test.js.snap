// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="col-lg-4"
>
  <div
    className="checkList"
  >
    <div
      className="checkListLogo"
    >
      <img
        className="catLogo"
        src="catLogo.png"
      />
    </div>
    <h2>
      Cat Adoption Checklist
    </h2>
    <p
      className="checkListText"
    >
      Go to Checklist
    </p>
    <a
      href="denemeLink"
    >
      <button
        className="checkListButton"
      >
        Read More
      </button>
    </a>
  </div>
</div>
`;
